### DISCUSSION TITLE
How to change the color of the scroll bar

### DISCUSSION THREAD

**Discussion by wangnan789** [tsujan/FeatherPad#767] [Q&A]
How to change the color of the scroll bar on the right, it and the background color are white, so that it is difficult to distinguish

**Comment by tsujan** [comment]
How scroll-bars/buttons/sliders/windows… are drawn as well as their colors are determined by the active Qt style, not by apps.

Either change your Qt style or change its palettes. Of course, you need a Qt config tool for that. KDE and LXQt provide config dialogs for Qt5 apps (until they're ported to Qt6, soon); in other DEs, `qt5ct` and `qt6ct` are often used, the latter being for Qt6 apps (if FeatherPad is compiled against Qt6).

**Comment by wangnan789** [comment]
I failed to change the color using qt6ct. I installed qt5-style-kvantum and qt5-style-kvantum-themes, used it to change the theme, changed the color

**Reply by tsujan** [reply]
Yes, Kvantum has many themes, and you might find one that suits your needs.

> I failed to change the color using qt6ct.

If FeatherPad is compiled against Qt5 in your distro, you'll need `qt5ct`. However, qt[x]ct's color management is mostly for Fusion (Qt's default style) and under DE's without a Qt config tool of their own. qt[x]ct can't affect Kvantum, and it can change Breeze's colors only partially.
